source/blog/2016-02-24-devconf-2016.html.md

```diff
@@ -0,0 +1,42 @@
+- Attended [Debugging the Virtualization Layer (libvirt and QEMU) in OpenStack - Kashyap Chamarthy](https://www.youtube.com/watch?v=Dd2AGGMWXQM "Debugging the Virtualization Layer (libvirt and QEMU) in OpenStack - Kashyap Chamarthy") - I think (and asked) if libvirt can do a better job - in saving in a cyclic log the QMP commands sent and the response from the guest and dump it in case of a guest crash.
+- Paid a visit to [Cockpit Hackfest - Dominik Perpeet, Marius Vollmer, Peter Volpe, Stef Walter](https://www.youtube.com/watch?v=TNDe90WSZow "Cockpit Hackfest - Dominik Perpeet, Marius Vollmer, Peter Volpe, Stef Walter") - as we intend to use Cockpit as our UI for oVirt Next Generation Node, it was great to present our use case and exchange thoughts, ideas and directions. Few bugs were filed during the hackfest per our comments.
+- Attended [Dockerizing JBoss Products - David Becvarik](https://www.youtube.com/watch?v=NpyEoFlDzOQ "Dockerizing JBoss Products - David Becvarik"). Great work done by the JBoss team to improve and streamline packaging of JBoss into containers.
+- Attended [Upstream First Testing - Tim Flink](https://www.youtube.com/watch?v=15-yXOJuonQ "Upstream First Testing - Tim Flink"). I discussed with the presenter the possibility to run oVirt with [Lago project on GitHub](https://github.com/lago-project/lago "Lago project on GitHub") in Fedora, in a CI way, by monitoring distgit changes for relevant packages (such as lvm2, device-mapper*, libvirt and others.
+- Attended [High performance VMs in OpenStack - Nikola Dipanov](https://www.youtube.com/watch?v=9J_sEdlpIlQ "High performance VMs in OpenStack - Nikola Dipanov") - looks like oVirt is already doing a lot of what OpenStack is working on to achieve high-performance from KVM!
+- Preached on running Lago with OpenStack, Gluster and Cockpit. At least the first two items look really promising and I'm looking forward to a collaboration in those efforts. I might just try to do the Gluster one myself.
+- [Avocado and Jenkins: Test Automation and CI - Lukáš Doktor, Yash Mankad](https://www.youtube.com/watch?v=rqavfmPAt7o "Avocado and Jenkins: Test Automation and CI - Lukáš Doktor, Yash Mankad") was interesting for me, as Avocado seems like a cool, mature testing framework - which I hope we can use in the future in CI testing.
+- Having used the [oVirt Live](http://www.ovirt.org/download/ovirt-live/"http://www.ovirt.org/download/ovirt-live/") USB DoK image in both FOSDEM and DevConf, I've found several items where we can improve in and filed relevant RFEs for them [1][2].
```

Comment:
Please rework the links into the sentence like you did with the others, so you can for example write "filed RFE <number> and <number2>".